weed/pb/volume_server.proto

```diff
@@ -147,6 +147,7 @@ message VacuumVolumeCompactRequest {
 }
 message VacuumVolumeCompactResponse {
     int64 processed_bytes = 1;
+    uint32 percent_load_avg_1m = 2;
```

Comment:
use "float64 load_avg_1m"

percent is how the value should be rendered.

Reply:
I thought about it, but float64 is definitely not suitable, since you need an integer from 0 to 100 with mathematical rounding